Makefile

```diff
@@ -26,6 +26,7 @@ infection: composer database
 	vendor/bin/infection
 
 integration: composer database
+	if [ ! -f "config/testing.yml" ]; then cp config/testing.dist.yml config/testing.yml; fi
```

Comment:
@mdwheele 

Added this line to `Makefile` to ensure the configuration file exists.

/cc @BackEndTea